src/frontends/onnx/frontend/include/onnx_import/core/operator_set.hpp

```diff
@@ -8,17 +8,16 @@
 #include <string>
 #include <unordered_map>
 
-#include "ngraph/node.hpp"
 #include "onnx_import/core/node.hpp"
 
-namespace ngraph {
+namespace ov {
```

Comment:
can we keep this stuff in old API only?

Reply:
Yes, I've reverted those changes after a chat with Ilya C.